profiles/kentik_snmp/cisco/cisco-all-devices.yml

```diff
@@ -1168,3 +1168,7 @@ metrics:
         column:
           name: cafSessionTimeout
           OID: 1.3.6.1.4.1.9.9.656.1.4.1.1.15
+      - tag: ifDescr
+        column: 
+          name: ifDescr
+          OID: 1.3.6.1.2.1.2.2.1.2
```

Comment:
@aditdeshpande10 - have you confirmed that the `ifIndex` value from the [ifTable](https://oidref.com/1.3.6.1.2.1.2.2.1.1) are mapped to the same values in the [cafPortConfigTable](https://oidref.com/1.3.6.1.4.1.9.9.656.1.2.1)? 

This would be the primary key the data is joined on, if this doesn't match up your intentions here won't be met.

Reply:
Yes @thezackm, I've confirmed that the values are the same. Index is only printed for the interfaces (physical) where 802.1x is enabled.
I'm attaching a snippet of the snmpwalk for your reference -

[user@host ~]$ snmpwalk -v3 <> arn3-axs-e 1.3.6.1.4.1.9.9.656.1.2.1.1.6

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.9 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.10 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.11 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.12 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.13 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.14 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.15 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.16 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.17 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.18 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.19 = INTEGER: 1

SNMPv2-SMI::enterprises.9.9.656.1.2.1.1.6.20 = INTEGER: 1


[user@host ~]$ snmpwalk -v3 <> arn3-axs-e 1.3.6.1.2.1.2.2.1.2

IF-MIB::ifDescr.1 = STRING: GigabitEthernet0/0

IF-MIB::ifDescr.2 = STRING: Bluetooth0/4

IF-MIB::ifDescr.3 = STRING: Null0

IF-MIB::ifDescr.4 = STRING: unrouted VLAN 1

IF-MIB::ifDescr.5 = STRING: unrouted VLAN 1002

IF-MIB::ifDescr.6 = STRING: unrouted VLAN 1004

IF-MIB::ifDescr.7 = STRING: unrouted VLAN 1005

IF-MIB::ifDescr.8 = STRING: unrouted VLAN 1003

IF-MIB::ifDescr.9 = STRING: TwoGigabitEthernet1/0/1

IF-MIB::ifDescr.10 = STRING: TwoGigabitEthernet1/0/2

IF-MIB::ifDescr.11 = STRING: TwoGigabitEthernet1/0/3

IF-MIB::ifDescr.12 = STRING: TwoGigabitEthernet1/0/4

IF-MIB::ifDescr.13 = STRING: TwoGigabitEthernet1/0/5

IF-MIB::ifDescr.14 = STRING: TwoGigabitEthernet1/0/6

IF-MIB::ifDescr.15 = STRING: TwoGigabitEthernet1/0/7

IF-MIB::ifDescr.16 = STRING: TwoGigabitEthernet1/0/8

IF-MIB::ifDescr.17 = STRING: TwoGigabitEthernet1/0/9

IF-MIB::ifDescr.18 = STRING: TwoGigabitEthernet1/0/10

IF-MIB::ifDescr.19 = STRING: TwoGigabitEthernet1/0/11

IF-MIB::ifDescr.20 = STRING: TwoGigabitEthernet1/0/12